README.md

```diff
@@ -347,7 +357,7 @@ def get_all_alerts():
 print(get_all_alerts())
 ```
 
-> This script is available in the `get_all_alerts.py` file.
+> This script improved by direct usage of SecureX credential is available in the `get_all_alerts.py` file. 
```

Comment:
`More complete version of this script is available in the `get_all_alerts.py` file.`